Improbable/Schema/Improbable.Schema.Bundle/TypeDescription.cs

```diff
@@ -42,7 +42,8 @@ public TypeDescription(string qualifiedName, Bundle bundle)
 
             IsRestricted = qualifiedName.StartsWith("improbable.restricted");
 
-            NestedEnums = bundle.Enums.Where(e => e.Value.OuterType == qualifiedName).Select(type => bundle.Enums[type.Key]).ToList();
+            NestedEnums = bundle.Enums.Where(e => e.Value.OuterType == qualifiedName)
```

Comment:
Might just be my preference, but I usually put every part of the LINQ statement on a new line.